pkg/sfu/videolayerselector/dependencydescriptor.go

```diff
@@ -139,6 +143,7 @@ func (d *DependencyDescriptor) Select(extPkt *buffer.ExtPacket, _layer int32) (r
 			continue
 		}
 
+		// DD-TODO : if activeDecodeTargets is nil, use last seen activeDecodeTargets
```

Comment:
Can you also add a comment on what is the TODO here @cnderrauber ? This comment seems to indicate current behaviour. If there is something else TODO for a future version, would be good to have what the intended TODO is.

Reply:
It is not the current behavior that we only check the packet's `ActiveDecodeTargetsBitmask` now, the publisher can send packets without the mask if `ActiveDecodeTargets` is not changed.